neuralmonkey/readers/plain_text_reader.py

```diff
@@ -62,7 +62,7 @@ def reader(files: List[str]) -> Iterable[List[str]]:
         for line in lines(files):
             if not line:
                 yield []
-            line = line.rstrip("\n")
+            line = line.rstrip("\r\n")
```

Comment:
Pokud řeknem že chcem zahazovat jakýkoliv trailing whitespace, tak bych tam dal jen `rstrip()` bez parametrů..

Reply:
Taky pozor, tadyten argument `"\r\n"` je vnímanej pythonem jako pole, takže by to mělo bejt spíš `["\r", "\n"]` aby bylo jasný, že kdyby náhodou nějakej string končil `"\n\r"` (vim, smysl to moc nedává), tak by se to taky useklo

Reply:
take se primlouvam za zahazovani cehokoliv napravo, tak to dam prazdne

Reply:
navic se to chova jinak nex string_reader ktery dela .strip() ... tak co to sjednotit a udelat i levy strip tady?

Reply:
jop, fouknul bych tam normalni strip()